packages/cli/src/eventbus/eventBus.controller.ts

```diff
@@ -68,44 +73,65 @@ const isMessageEventBusDestinationOptions = (
 };
 
 // ----------------------------------------
-// Events
+// Controller
 // ----------------------------------------
-eventBusRouter.get(
-	'/event',
-	ResponseHelper.send(async (req: express.Request): Promise<any> => {
+
+@RestController('/eventbus')
+export class EventBusController {
+	// ----------------------------------------
+	// Events
+	// ----------------------------------------
+	@Get('/event', { middlewares: [isOwnerMiddleware] })
+	async getEvents(req: express.Request): Promise<any> {
+		let result: EventMessageTypes[] | Record<string, EventMessageTypes[]> = [];
 		if (isWithQueryString(req.query)) {
 			switch (req.query.query as EventMessageReturnMode) {
 				case 'sent':
-					return eventBus.getEventsSent();
+					result = await eventBus.getEventsSent();
+					break;
 				case 'unsent':
-					return eventBus.getEventsUnsent();
+					result = await eventBus.getEventsUnsent();
+					break;
 				case 'unfinished':
-					return eventBus.getUnfinishedExecutions();
+					result = await eventBus.getUnfinishedExecutions();
+					break;
 				case 'all':
 				default:
-					return eventBus.getEventsAll();
+					result = await eventBus.getEventsAll();
 			}
+		} else {
+			result = await eventBus.getEventsAll();
 		}
-		return eventBus.getEventsAll();
-	}),
-);
+		return result;
+	}
+
+	@Get('/failed')
+	async getFailedEvents(req: express.Request): Promise<FailedEventSummary[]> {
+		let result: FailedEventSummary[] = [];
+		let amount = 5;
+		try {
+			amount = parseInt(req.query?.amount as string);
```

Comment:
since `req.query` will always be an object, this line will never throw. instead you'd end up with `amount` = `NaN` for invalid values of `amount`